lib/index.js

```diff
@@ -309,4 +324,236 @@ module.exports = {
 
     return debugTree(new MergeTrees([ember, templateCompiler, jquery]), 'vendor:final');
   },
+
+  _issueGlobalsDeprecation() {
```

Comment:
This function looks quite complex, but I don't see any tests ensuring it is both accurate and remains accurate overtime. Are these present elsewhere or is this an oversight?